Nightly failed on phpStan-php 7.4

```
 ------ ----------------------------------------------------- 
  Line   lib/Importer/FilesImporter.php                       
 ------ ----------------------------------------------------- 
  94     Cannot call method getPath() on object|true.         
  101    Cannot call method getType() on object|true.         
  119    Cannot call method getETag() on object|true.         
  120    Cannot call method getPermissions() on object|true.  
  126    Cannot call method getType() on object|true.         
  127    Cannot call method getPath() on object|true.         
  133    Cannot call method getETag() on object|true.         
  134    Cannot call method getPermissions() on object|true.  
 ------ ----------------------------------------------------- 

 ------ ---------------------------------------------------- 
  Line   lib/Importer/MetadataImporter/ShareImporter.php     
 ------ ---------------------------------------------------- 
  136    Cannot call method getShareType() on object|true.   
  138    Cannot call method getOwner() on object|true.       
  140    Cannot call method getSharedWith() on object|true.  
  146    Cannot call method getPath() on object|true.        
  147    Cannot call method getOwner() on object|true.       
  154    Cannot call method getOwner() on object|true.       
  155    Cannot call method getSharedWith() on object|true.  
  160    Cannot call method getPath() on object|true.        
  161    Cannot call method getSharedWith() on object|true.  
  167    Cannot call method getPath() on object|true.        
  168    Cannot call method getOwner() on object|true.       
  174    Cannot call method getOwner() on object|true.       
  179    Cannot call method getPath() on object|true.        
  180    Cannot call method getOwner() on object|true.       
  186    Cannot call method getOwner() on object|true.       
  187    Cannot call method getSharedWith() on object|true.  
  197    Cannot call method getPath() on object|true.        
  198    Cannot call method getOwner() on object|true.       
 ------ ---------------------------------------------------- 

 ------ ----------------------------------------------------------- 
  Line   lib/Importer/TrashBinImporter.php                          
 ------ ----------------------------------------------------------- 
  124    Cannot call method getPath() on object|true.               
  125    Cannot call method getPath() on object|true.               
  131    Cannot call method getType() on object|true.               
  149    Cannot call method getETag() on object|true.               
  150    Cannot call method getPermissions() on object|true.        
  155    Cannot call method getType() on object|true.               
  158    Cannot call method getETag() on object|true.               
  159    Cannot call method getPermissions() on object|true.        
  164    Cannot call method getOriginalLocation() on object|true.   
  166    Cannot call method getOriginalName() on object|true.       
  168    Cannot call method getDeletionTimestamp() on object|true.  
  169    Cannot call method getOriginalLocation() on object|true.   
 ------ ----------------------------------------------------------- 

 [ERROR] Found 38 errors                                                        

make: *** [Makefile:143: test-php-phpstan] Error 1
```

his prevents any later tests from being run.

--- a/lib/Importer/FilesImporter.php
+++ b/lib/Importer/FilesImporter.php
@@ -91,6 +91,9 @@ class FilesImporter {
 			)
 				!== false
 			) {
+				/**
+				 * @var File $fileMetadata
+				 */
 				$fileCachePath = $fileMetadata->getPath();
 				$pathToFileInExport = Path::join($exportRootFilesPath, $fileCachePath);
 

--- a/lib/Importer/MetadataImporter/ShareImporter.php
+++ b/lib/Importer/MetadataImporter/ShareImporter.php
@@ -133,6 +133,9 @@ class ShareImporter {
 				)
 			) !== false
 			) {
+				/**
+				 * @var Share $shareModel
+				 */
 				switch ($shareModel->getShareType()) {
 					case Share::SHARETYPE_USER:
 						if ($shareModel->getOwner() === $userId) {

--- a/lib/Importer/TrashBinImporter.php
+++ b/lib/Importer/TrashBinImporter.php
@@ -121,6 +121,9 @@ class TrashBinImporter {
 			)
 				!== false
 			) {
+				/**
+				 * @var TrashBinFile $fileMetadata
+				 */
 				$fileCachePath = Path::join('/files_trashbin/files/', $fileMetadata->getPath());
 				$pathToFileInExport = Path::join($exportRootFilesPath, $fileMetadata->getPath());
 

